### DISCUSSION TITLE
Text Generator + Canvas

### DISCUSSION THREAD

**Discussion by nhaouari** [nhaouari/obsidian-textgenerator-plugin#75] [Show and tell]
AI brainstorming through Canvas and Text Generator Plugin. 
Currently in private beta, but stay tuned for public availability.

What do you think ? 

https://user-images.githubusercontent.com/9850722/210185961-11217e46-68db-42ac-861f-d35d27607e37.mp4

**Comment by arminta7** [comment]
This looks amazing! Can we join the private beta?

**Reply by ObsidianPC** [reply]
Yes please - this is something I would love to try out and give feedback on

**Comment by lem447** [comment]
This is completely game changing

**Comment by raphael-kagermeier** [comment]
That's awesome, may I join the private beta?
I might also be able to contribute!

**Comment by georgepor** [comment]
I've been longing for something like this since I heard of the Canvas and started playing with ChatGPT on Nov 30 but not being a geek, all I could do was only longing. Please invite me to play with it as soon as you can!

**Reply by ricraftis** [reply]
Indeed George. Looks awesome and has great potential to create concepts for discussion and generation thinking.

**Comment by nirvana6** [comment]
Just tell me how much money to experience it in advance?

**Comment by dvansoye** [comment]
Amazing!

**Comment by geschenkwald** [comment]
I would love to join the private beta!

**Comment by mycosavant** [comment]
please? 😺

**Comment by Franznnn** [comment]
would that be possible? I be highly interested, would provide feedback as well...

**Comment by AMaestroG** [comment]
Esto es lo más interesante antes incluso de la salida chatgpt5, ¿cómo puedo entrar en al Beta?

**Comment by mounta11n** [comment]
At a least a response would be really nice from you @nhaouari !

**Comment by wwboynton** [comment]
I am also interested in this, and have opened a [thread asking about similar functionality in the discord](https://discord.com/channels/1083485983879741572/1166798088560726106):

**Comment by CodeOfficer** [comment]
Amazing .... I wanna try it!
